src/ui/ResultLink/ResultLink.ts

```diff
@@ -360,6 +355,19 @@ export class ResultLink extends Component {
     );
   }
 
+  private getDisplayedTitle(result?: IQueryResult) {
```

Comment:
Is the `result` argument being used inside this function?

Reply:
You're right, it isn't. I misinterpreted the code I moved over. I'm going to remove this parameter.